[2023-06-01T11:23:20.307+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-06-01T11:23:20.342+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2023-06-01T11:23:20.349+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:23:20.351+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:23:20.352+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:23:20.406+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-05-30 00:00:00+00:00
[2023-06-01T11:23:20.429+0000] {standard_task_runner.py:55} INFO - Started process 1678 to run task
[2023-06-01T11:23:20.454+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '532', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpns839qk8']
[2023-06-01T11:23:20.456+0000] {standard_task_runner.py:83} INFO - Job 532: Subtask insert_into_table
[2023-06-01T11:23:20.654+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-05-30T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:23:20.954+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2023-06-01T11:23:20.958+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:23:20.986+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:23:21.506+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:23:21.538+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-05-30', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:23:21.548+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:23:21.577+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220530T000000, start_date=20230601T112320, end_date=20230601T112321
[2023-06-01T11:23:21.657+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:23:21.734+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
